packages/ui/src/input/index.tsx

```diff
@@ -76,6 +84,30 @@ const Input = forwardRef<HTMLInputElement, InputProps>(function (
         }}
         {...props}
       />
+      {type === 'password' && !isVisible && (isFocused || value) && (
+        <EyeCross
+          onClick={onVisible}
+          fill="white"
+          className="absolute"
+          style={{
+            top: '50%',
+            right: '20px',
+            transform: 'translate(-50%, -50%)',
+          }}
+        />
+      )}
+      {type === 'password' && isVisible && (isFocused || value) && (
+        <Eye
+          onClick={onVisible}
+          fill="white"
+          className="absolute"
+          style={{
+            top: '50%',
+            right: '20px',
+            transform: 'translate(-50%, -50%)',
+          }}
+        />
+      )}
```

Comment:
```suggestion
      <If condition={type === 'password' && (isFocused || value)}>
         <Choose value={isVisible}>
            <When value={true}>
               <EyeCross
                 onClick={onVisible}
                 fill="white"
                 className="absolute"
                 style={{
                 top: '50%',
                 right: '20px',
                 transform: 'translate(-50%, -50%)',
               }}
              />
            </When>
            <Otherwise>
               <Eye
                 onClick={onVisible}
                 fill="white"
                 className="absolute"
                 style={{
                    top: '50%',
                    right: '20px',
                    transform: 'translate(-50%, -50%)',
                 }}
               />
            </Otherwise>
         </Choose>
      </If>
```